packages/expo-sqlite/src/next/NativeDatabase.ts

```diff
@@ -48,4 +48,11 @@ export interface OpenOptions {
    * @default false
    */
   useNewConnection?: boolean;
+
+  /**
+   * Finalized unclosed statements automatically when the database is closed.
+   * @default true
+   * @hidden
```

Comment:
I guess it's intentionally hidden, right?

Reply:
right! it's nice to have the feature enabled and i don't want to document it 😂